modules/nw-dns-forward.adoc

```diff
@@ -5,7 +5,16 @@
 [id="nw-dns-forward_{context}"]
 = Using DNS forwarding
 
-You can use DNS forwarding to override the forwarding configuration identified in `/etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone. If the forwarded zone is the Ingress domain managed by {product-title}, then the upstream name server must be authorized for the domain.
+You can use DNS forwarding to:
```

Comment:
From IBM Style Guide (ISG): 

> Generally, use a sentence to introduce a list that is not a procedure.

I suggest something like this:
```suggestion
You can use DNS forwarding to change configuration settings in the following ways:
```

Reply:
Comment addressed.